libs/newsletter/guides/Overview.md

```diff
@@ -0,0 +1,68 @@
+# Daffodil Newsletter
+
+The Daffodil Newsletter library manages your newsletter subscription service. It utilizes multiple different ecommerce API drivers to make hooking in your app's newsletter simple while also providing redux state management for your UI. 
+
+# Getting Started
+To get started import the `StoreModule` and the `DaffNewsletter` at the top of your app.module file.
+```typescript
+import { DaffNewsletterModule } from '@daffodil/newsletter';
+import { StoreModule } from '@ngrx/store;'
+```
+Then import the `DaffNewsletterModule` in your app.module. Afterwards, also import `StoreModule.forRoot({})`, as this will be relevant later when utilizing the redux features of the newsletter module.
+
+```typescript
+@ngModule({
+  imports:[
+    DaffNewsletterModule,
+    StoreModule.forRoot({})
+  ]
+})
+```
+Then to utilize the Newsletter Facade inside your component, include an instance of `DaffNewsletterFacade` in your component's constructor.
```

Comment:
Whoa, what's a facade? That's a weird word, can we explain either here or in another doc? We should also try to clarify its purpose.